package.json

```diff
@@ -29,8 +33,16 @@
         "html-webpack-plugin": "^5.5.0",
         "image-webpack-loader": "^8.1.0",
         "jest": "^27.4.7",
+        "mini-css-extract-plugin": "^2.5.3",
+        "postcss-loader": "^6.2.1",
+        "sass": "^1.49.0",
+        "sass-loader": "^12.4.0",
+        "style-loader": "^3.3.1",
+        "stylelint": "^14.3.0",
```

Comment:
I'm not seeing this in webpack config or as a command in `package.json`. Is this an oversight, or is the intent here that it's just used for an editor plugin?